internal/pkg/panic/panic.go

```diff
@@ -0,0 +1,88 @@
+package panic
+
+import (
+	"fmt"
+	"runtime/debug"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"golang.org/x/exp/slices"
+
+	cliv1 "github.com/confluentinc/ccloud-sdk-go-v2/cli/v1"
+
+	v1 "github.com/confluentinc/cli/internal/pkg/config/v1"
+	"github.com/confluentinc/cli/internal/pkg/usage"
+)
+
+type Panic struct {
+	ErrorMsg string
+}
+
+func (p *Panic) Error() string {
+	return p.ErrorMsg
+}
+
+// CollectPanic collects relevant usage data for when panics occur and command execution is not completed.
+func CollectPanic(cmd *cobra.Command, args []string, cfg *v1.Config) *usage.Usage {
+	u := usage.New(cfg.Version.Version)
+	fullCommand, flags, _ := cmd.Find(args)
+	u.Command = cliv1.PtrString(fullCommand.CommandPath())
+	u.Flags = parseFlags(fullCommand, flags)
+	u.Error = cliv1.PtrBool(true)
+	u.StackFrames = parseStack(string(debug.Stack()))
+	return u
```

Comment:
Let's use the following style here?
```suggestion
	return &usage.Usage{
		Command: ...,
	}
```